kornia/augmentation/container/augment.py

```diff
@@ -469,6 +469,52 @@ def forward(  # type: ignore[override]
 
         return outputs
 
+    def __call__(
+        self,
+        *inputs: Any,
+        input_names_to_handle: Optional[List[Any]] = None,
+        output_type: str = "tensor",
+        **kwargs: Any,
+    ) -> Any:
+        """Overwrites the __call__ function to handle various inputs.
+
+        Args:
+            input_names_to_handle: List of input names to convert, if None, handle all inputs.
+            output_type: Desired output type ('tensor', 'numpy', or 'pil').
+
+        Returns:
+            Callable: Decorated function with converted input and output types.
+        """
+
+        # Wrap the forward method with the decorator
+        if not self._disable_features:
+            # TODO: Some more behaviour for AugmentationSequential needs to be revisited later
+            # e.g. We convert only images, etc.
+            decorated_forward = self.convert_input_output(
+                input_names_to_handle=input_names_to_handle, output_type=output_type
+            )(super(ImageSequential, self).__call__)
+            _output_image = decorated_forward(*inputs, **kwargs)
+
+            if len(inputs) == 1 and isinstance(inputs[0], dict):
+                original_keys, in_data_keys, inputs, invalid_data = self._preproc_dict_data(inputs[0])
+            else:
+                in_data_keys = kwargs["data_keys"] if "data_keys" in kwargs else self.data_keys
+            data_keys = self.transform_op.preproc_datakeys(in_data_keys)
+
+            if len(data_keys) > 1 and data_keys.index(DataKey.INPUT):
```

Comment:
Shouldn't this be checking if `DataKey.INPUT` is in data_keys? According to this condition, we only enter the if block if `len(data_keys) > 1` and if INPUT is not the first datakey.